src/components/disappearing-header/index.tsx

```diff
@@ -284,6 +292,13 @@ const useThemeStyles = StyleSheet.createThemeHook((theme) => ({
     right: 0,
   },
 
+  highlightComponent: {
```

Comment:
Is there a possibility that font scaling up to 200% and/or a little more text content to the higlightComponent could possibly render off screen and become inaccessible? 
Just wanted to mention this, as we had those issues on the ticketing splash screen and onboarding views. We solved it by making the contents scrollable

Reply:
I'll look into it 👍

Reply:
Yeah, lots of text and font scaling did not look too good. Talked with Petter about this, and he was not a big fan of scrolling.

Proposed solution:
- We set a description in Firebase Remote Config that specifies the maximum length for the text field
![Screenshot 2020-11-27 at 10 41 11](https://user-images.githubusercontent.com/675421/100434648-1a0d2580-309d-11eb-92ef-6b8fee8cfc39.png)
- A text length of up to 120 characters and link text length of up to 40 characters will fit with font size set to max on both iOS and Android, also on small screens. This is tested locally.
- Overflowing text will be hidden. This should not happen unless someone do not follow the descriptions in Remote Config.

Does this sound as a good enough solution?

Reply:
After testing with the maximum accessibility text size in iPhone settings we saw that 120 characters was actually too many to be presented without overflowing. We ended up adding ScrollView, for the cases where both the number of characters and accessibility text size is maxed out. We preserve the text length descriptions in Firebase Remote Config so scrolling rarely have to be used.